src/main/java/org/sonar/plugins/stash/StashPlugin.java

```diff
@@ -124,6 +125,14 @@ public void define(Context context) {
                           .subCategory(CONFIG_PAGE_SUB_CATEGORY_STASH)
                           .onQualifiers(Qualifiers.PROJECT)
                           .defaultValue(DEFAULT_STASH_THRESHOLD_VALUE).build(),
+        PropertyDefinition.builder(STASH_ISSUE_SEVERITY_THRESHOLD)
+                          .name("Stash issue severity Threshold")
+                          .description("Only push issues with the same or higher severity to Stash server")
+                          .type(PropertyType.SINGLE_SELECT_LIST)
+                          .subCategory(CONFIG_PAGE_SUB_CATEGORY_STASH)
+                          .onQualifiers(Qualifiers.PROJECT)
+                          .defaultValue(Severity.INFO)

```

Comment:
Personally I would prefer to use `SEVERITY_LIST.get(0)` as the default value, what is your opiniion?

Reply:
Good idea, should there be a lower severity added in the future, it'll still pick the lowest one properly.